pkg/apis/resource/v1alpha1/storage/artifact_bucket.go

```diff
@@ -1,103 +0,0 @@
-/*
-Copyright 2019-2020 The Tekton Authors
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package storage
-
-import (
-	"fmt"
-
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline"
-	"github.com/tektoncd/pipeline/pkg/apis/pipeline/v1beta1"
-	resource "github.com/tektoncd/pipeline/pkg/apis/resource/v1alpha1"
-	"github.com/tektoncd/pipeline/pkg/names"
-	corev1 "k8s.io/api/core/v1"
-)
-
-// For some reason gosec thinks this string has enough entropy to be a potential secret.
-// The nosec comment disables it for this line.
-/* #nosec */
-const secretVolumeMountPath = "/var/bucketsecret"
-
-// ArtifactBucket contains the Storage bucket configuration defined in the
-// Bucket config map.
-// +k8s:deepcopy-gen=true
-type ArtifactBucket struct {
-	Name     string
-	Location string
-	Secrets  []resource.SecretParam
-
-	ShellImage  string
-	GsutilImage string
-}
-
-// GetType returns the type of the artifact storage
-func (b *ArtifactBucket) GetType() string {
-	return pipeline.ArtifactStorageBucketType
-}
-
-// StorageBasePath returns the path to be used to store artifacts in a pipelinerun temporary storage
-func (b *ArtifactBucket) StorageBasePath(pr *v1beta1.PipelineRun) string {
```

Comment:
It seems that the removal of `Storage` resources would lead to the functionalities of artifiact_bucket, it has to be handled for the pipelinerun reconciler.